
sensor_msgs/Image image
float32[] bbox # xmin, xmax, ymin, ymax
sensor_msgs/Image heatmap


